src/key_stores/unencrypted_file_system_keystore.ts

```diff
@@ -57,6 +58,11 @@ export class UnencryptedFileSystemKeyStore extends KeyStore {
 
     constructor(keyDir: string) {
         super();
+        // Check if path is absolute by resolving it and comparing to normailzed path.
+        if (path.resolve(keyDir) !== path.normalize(keyDir)) {
+            // If path is not absolute, add current working directory.
+            keyDir = path.join(process.cwd(), keyDir);
```

Comment:
why not just use `path.resolve`?  You can shorten whole thing to:

```
this.keyDir = path.resolve(process.cwd(), keyDir);
```

it will automatically do the right thing for `keyDir` being absolute:
https://nodejs.org/api/path.html#path_path_resolve_paths